core/domain/state_domain.py

```diff
@@ -3458,6 +3578,51 @@ def convert_html_fields_in_state(
 
         return state_dict
 
+    @classmethod
+    def update_image_sizes_in_bytes_in_state(
```

Comment:
Method 2:
Having a separate method `update_image_sizes_in_bytes_in_state`, which is going to be called in the conversion function for states dict from v47 to v48.

Pros:
- This method is simpler, because, we will not need to make any changes to the existing `convert_html_fields_in_state` (i.e. we have to pass entity_id, entity_type as additional parameters)